packages/block-library/src/image/edit.js

```diff
@@ -165,11 +169,9 @@ class ImageEdit extends Component {
 			} );
 			return;
 		}
-
```

Comment:
I liked the whitespace! 😢

Reply:
Ah, ok! It's not the end of the world, but might be good to add it back if it was unintentional. There were a few others as well.

Reply:
Not a problem it should be all set now - thanks for pointing it out

Reply:
@talldan Replacing that whitespace triggered Travis CI and it failed for some unrelated reason 🤦‍♂️  
Is that something you can restart, or should I ask somewhere else?

Reply:
Hi @brentswisher - there were some minor issues after the 5.0 release with the CI jobs. Everything now seems to be passing so will merge this 👍

Reply:
Awesome, thanks!